test/e2e/specs/specs-playwright/wp-likes__comment-spec.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import {
+	DataHelper,
+	LoginFlow,
+	MyHomePage,
+	PublishedPostsListPage,
+	CommentsComponent,
+	CommentsLikesComponent,
+	GutenbergEditorPage,
+	NewPostFlow,
+} from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const quote =
+	'The foolish man seeks happiness in the distance. The wise grows it under his feet.\n— James Oppenheim';
+
+describe( DataHelper.createSuiteTitle( 'Likes (Comment) ' ), function () {
+	describe( 'Comment and like on an existing post', function () {
+		let commentsLikesComponent;
+
+		it( 'Log in', async function () {
+			const loginFlow = new LoginFlow( this.page, 'gutenbergSimpleSiteUser' );
+			await loginFlow.login();
+		} );
+
+		it( 'Visit site', async function () {
+			const myHomePage = await MyHomePage.Expect( this.page );
+			await myHomePage.visitSite();
```

Comment:
There's the [`Visit` static method in the ole `AsyncBaseContainer`](https://github.com/Automattic/wp-calypso/blob/e41c8158b83e89036cd1c6ccd313d7ea02ab17e0/test/e2e/lib/async-base-container.js#L34-L42). Do you think we should also have one for the new `BaseContainer`?

Reply:
I was not a fan of the `Visit` static method, but can't quite communicate why I feel that way. 

It's something I am open to implementing. Will file a task on the board.

Reply:
Task filed as #53601 :)